federatedscope/cl/fedgc/server.py

```diff
@@ -0,0 +1,251 @@
+import torch
+import logging
+import copy
+import numpy as np
+
+from federatedscope.core.message import Message
+from federatedscope.core.workers.server import Server
+from federatedscope.core.auxiliaries.utils import merge_dict
+from federatedscope.cl.fedgc.utils import global_NT_xentloss
+from torchviz import make_dot, make_dot_from_trace
```

Comment:
Please add `torchviz` to the dependency [cl], and I can't run CL with the minimal version of FS.

Reply:
it's deleted in new pr